doc_src/functional_hazard_analysis_longline/fha_longline.tex

```diff
@@ -355,34 +358,847 @@ \subsection{Phase 1: Approach and Positioning}
+	     	\tabitem Minimize the possible fall height of the \textcolor{purple}{patient} in the event of a line tear},
+	     fhaCritPostKeyValue=
+	     	{\textcolor{orange}{medium}. Very unlikely if the anchors are cleaned and personnel at the anchors are careful.
+			In addition, the fall height can be kept small and the pilot can be prepared}]
+
+%2.8
+\fhaTable[fhaSystemKeyValue=
+			{Leash},
+		 fhaFunctionKeyValue=
+		 	{load transfer},
+	     fhaCritPreKeyValue=
+	     	{\textcolor{red}{high}},
+	     fhaNumberKeyValue=
+	     	{2.8},
+	     fhaCausesKeyValue=
+	     	{Fall of two people in leash when releasing the rope}, 
+	     fhaDescriptionKeyValue=
+	     	{If the rope has to be released during this phase, the \textcolor{teal}{rescuer} and \textcolor{purple}{patient} will fall into the patient's leash.}, 
+	     fhaMitigationKeyValue=
+	     	{\tabitem Avoid sharp objects on harnesses/clothing of \textcolor{teal}{rescuer} and \textcolor{purple}{patient}\\
+	     	\tabitem Cutting tool for the leash cut remains secured in a holder\\
+	     	\tabitem No unprotected, sharp objects on \textcolor{purple}{patient} and \textcolor{teal}{rescuer}},
+	     fhaCritPostKeyValue=
+	     	{\textcolor{red}{high}, but can easily be avoided by taking measures (use of suitable clothing and belts and appropriate caution).}]
+
+%2.9
+\fhaTable[fhaSystemKeyValue=
+			{belt},
+		 fhaFunctionKeyValue=
+		 	{load transfer},
+	     fhaCritPreKeyValue=
+	     	{\textcolor{orange}{medium}},
+	     fhaNumberKeyValue=
+	     	{2.9},
+	     fhaCausesKeyValue=
+	     	{Fall of two people in leash when releasing the rope}, 
+	     fhaDescriptionKeyValue=
+	     	{If the rope has to be released in this phase, the \textcolor{teal}{rescuer} and \textcolor{purple}{patient} fall into the patient's leash.
+	      	The rescuer's load path is via the sling into the belay loop of the patient's harness.
+	       	The leash is usually tied parallel to the belay ring (threaded through the loops of the harness, similarly to tying in a climbing rope).}, 
+	     fhaMitigationKeyValue=
+	     	{\tabitem Attach the sling to the leash attachment loop and to the patient harness attachment ring.\\
+	      	If only one is possible: attach it to the leash attachment loop.\\
+	     	\tabitem Checking that the \textcolor{purple}{patient} has correctly attached the sling\\},
+	     fhaCritPostKeyValue=
+	     	{\textcolor{green}{low}. Can be controlled very well through communication and the 4-eyes principle.\\
+ 			However, a suitable communication channel is a mandatory}]
+
+\clearpage
+\subsection{Phase 3: Lifting to relieve the highline}
+\label{sec:fha:phase3}
+
+This phase begins after the \textcolor{purple}{patient} has been successfully hooked into the sling and the \textcolor{teal}{rescuer} and \textcolor{purple}{patient} are lifted
+by the helicopter. The lifting is complete when the highline is (almost) free of load
```

Comment:
```suggestion
This phase begins after the \textcolor{purple}{patient} has been successfully connected into the longline and the \textcolor{teal}{rescuer} and \textcolor{purple}{patient} are lifted
by the helicopter. The lifting is complete when the highline is (almost) free of load
```

Reply:
done in https://github.com/JBludau/Highline-Rescue-Project/pull/10/commits/5ca40df93d1aa74036bfab5c501c1f91d8d9a008 for the whole document